baybe/parameters/enum.py

```diff
@@ -17,17 +19,39 @@ class CategoricalEncoding(ParameterEncoding):
     """Integer encoding."""
 
 
-class SubstanceEncoding(ParameterEncoding):
-    """Available encodings for substance parameters."""
-
-    MORDRED = "MORDRED"
-    """Encoding based on Mordred chemical descriptors."""
-
-    RDKIT = "RDKIT"
-    """Encoding based on RDKit chemical descriptors."""
-
-    MORGAN_FP = "MORGAN_FP"
-    """Encoding based on Morgan molecule fingerprints."""
+# TODO Ideally, this should be turned into a class that can:
+#  - return default when CHEM not installed
+#  - check if enum is fingerprint
+PARAM_SUFFIX_FINGERPRINT = "Fingerprint"
+
+if CHEM_INSTALLED:
+    import inspect
+
+    from baybe._optional.chem import BaseFingerprintTransformer, skfp_fingerprints
+
+    AVAILABLE_SKFP_FP = dict(
```

Comment:
I will likely drop the inspect.get members as it would directly resolve the comments re this enum class and it makes the code easier to check as currently enum is not literal, which is also an issue